src/views/v2/rsvp/actions/success.php

```diff
@@ -17,6 +17,13 @@
  */
 
 $toggle_id = 'toggle-rsvp-' . $rsvp->ID;
+$attendee_ids = '';
+$opt_in_nonce = '';
+
+if ( ! empty( $process_result['opt_in_args'] ) ) {
+	$attendee_ids = $process_result['opt_in_args']['attendee_ids'];
```

Comment:
Is there any way to move this logic to the template invoke and send them as parameters instead of doing it here? The idea is to leave the templates as clean as possible.

Reply:
Doing this